libr/core/cmd_print.c

```diff
@@ -5659,6 +5659,75 @@ l = use_blocksize;
 				}
 			}
 			break;
+		case 't': // "pst"
+			if (l > 0) {
+				ut8 *var = malloc (len);
+				size_t len_charset = sdb_count (core->anal->charset_db);
+				r_charset_encode_str (var, core->block, len, &core->custo, len_charset);
+				r_print_string (core->print, core->offset, var, len, R_PRINT_STRING_ZEROEND);
+				free(var);
+				}
+			break;
+		case 'e': // "pse"
+			if (input[2] == ' ') {
+				const char *fname = r_str_trim_head_ro (input + 3);
```

Comment:
move this logic into a separate function `cmd_pse (RCore *...)`

Reply:
Done. I will add an unit test next.